jwst/ramp_fitting/gls_fit.py

```diff
@@ -692,52 +681,44 @@ def gls_fit(ramp_data, input_var_data,
 
     # Divide by sqrt(2) to convert the readnoise from CDS to single readout.
     rn3d = readnoise.reshape((nz, 1, 1)) * SINGLE_READOUT_RN_FACTOR
-    cov += (I * (rn3d**2 / M))
+    ramp_cov += (I * (rn3d**2 / M))
```

Comment:
The factor 1 / sqrt(nframes) has already been applied by utils.get_ref_subs.

Reply:
division by M removed